assets/scss/base/form/_form.scss

```diff
@@ -0,0 +1,330 @@
+
+Styleguide Form.field
+*/
+
+.form-field {
+  @extend %contain-floats;
+  float: left;
+  margin-bottom: 0.75em;
+  margin-top: 0;
+  width: 100%;
+}
+
+.form-field--tall {
+  margin: 0 0 em(30) 0;
+}
+
+
+/*
+Inline
+
+For an inline elements you have a few choices.
+
+Markup:
+<fieldset class="form-field-group">
+  <div class="{{modifier_class}}">
+    <label for="blue" class="block-label">
+      <input type="radio" id="blue"/>
+      Blue
+    </label>
+  </div>
+  <div class="{{modifier_class}}">
+    <label for="green" class="block-label">
+      <input type="radio" id="green"/>
+      Green
+    </label>
+  </div>
+  <div class="form-field">
+    <button class="button" type="submit">Send</button>
+  </div>
+</fieldset>
+
+.form-field--inline              - For an inline form field use `.form-field--inline`
+.form-field--inline-text-input   - For an inline form field for text input use `.form-field--inline-text-input`
+.form-field--inline-spaced-right - For an inline form field for text input spaced to the right use `.form-field--inline-spaced-right`
+.form-field--inline-spaced-left  - For an inline form field for text input spaced to the left use `.form-field--inline-spaced-left`
+
+Styleguide Form.inline
+*/
+
+/**
+ *  Allows you to stack input blocks side by side
+ */
+.form-field--inline {
+  @extend .form-field;
+  clear: none;
+  float: left;
+  width: auto;
+  position: relative;
+
+  input[type="text"], input[type="password"] {
+    margin: 0 0.52em;
+  }
+}
+
+.form-field--inline-text-input {
+  @extend .form-field--inline;
+}
+
+.form-field--inline-spaced-right {
+  @extend .form-field--inline;
+  margin-right: 15px;
+}
+
+.form-field--inline-spaced-left {
+  @extend .form-field--inline;
+  margin-left: 15px;
+}
+
+
+/*
+Submit
+
+A class to provide extra space between the `submit` input and inputs found before.
+
+Markup:
+<div class="form-field form-field--submit">
+  <button class="button" type="submit">Send</button>
+</div>
+
+Styleguide Form.submit
+*/
+
+/**
+ * Wrap action elements of form (submit button, cancel link)
+ * to provide space between these and last inputs of form
+ */
+.form-field--submit {
+  margin-top: 15px;
+}
+
+
+/*
+Field hint
+
+A hint for a form field
+
+Markup:
+<p class="form-field--hint">Example hint</p>
+
+Styleguide Form.hint
+*/
+
+.form-field--hint {
+  @extend .faded-text;
+  display: block;
+  margin-bottom: 0.75em;
+}
+
+
+/*
+Date
+
+For displaying a date you can use the following markup.
+> note need for `.input--xsmall` class on input.
+
+Markup:
+<div class="form-date">
+  <div class="form-group form-group-year">
+    <label for="year" class="block-label">
+      <input class="input--xsmall" type="number" pattern="[0-9]*" min="0" max="31" id="year" name="year"/>
+      Blue
+    </label>
+  </div>
+</div>
+
+Styleguide Form.date
+*/
+
+
+.form-date {
+  @extend %contains-floats;
+  display: inline-block;
+
+  @include ie-lte(7){
+    zoom: 1;
```

Comment:
although just copied and pasted I do like the attention to detail on display here